api/src/main/java/mcoffin/rogue/api/EncoderComponent.java

```diff
@@ -0,0 +1,6 @@
+package mcoffin.rogue.api;
+
+public interface EncoderComponent {
+  public double getEncoderPosition();
```

Comment:
I now you said you think this should be returning a double, but I don't see how you can represent a 2-dimensional robot position relative to the last `resetEncoders` in one double. @WillYingling